builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -124,6 +129,12 @@ func resourceVSphereVirtualMachine() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"memory_reservation": &schema.Schema{
+				Type:     schema.TypeInt,
+				Optional: true,
+				ForceNew: true,
```

Comment:
Do you really want to force a new resource when memory_reservation changes? Can it not be changed once created?